doc/source/cluster/kubernetes/examples/verl-post-training.md

```diff
@@ -0,0 +1,150 @@
+(kuberay-verl)=
+# Reinforcement Learning with Human Feedback (RLHF) for LLMs with verl on KubeRay
+
+[verl](https://github.com/volcengine/verl) is an open-source framework that provides a flexible, efficient, and production-ready RL training library for large language models (LLMs).
```

Comment:
Just confirming: do all the acronyms/abbreviations in this first paragraph common knowledge for all readers? I'm not familiar with some of these terms, but also not as entrenched in this space.